Revision and user-friendly improvements (#2)
eProsima/SOSS-docs: docs/index.rst

    :alt: eProsima
    :target: http://www.eprosima.com/

> *SOSS* is a *System-Of-Systems Syntesizer* that allows communication among an arbitrary number of protocols that speak different languages.

Author: ```suggestion
*SOSS* is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that speak different languages.
```

> If one has for instance a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, SOSS can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A<->B bridge is configured with a text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.

Author: Split long lines (try to adjust them into 120 characters).

Reviewer: ok. see if you like it.

Author: The line is still too long. I don't mean by punctuation but literally the length of the line in the editor. You must follow the coding style (max. 120 chars per line).

Reviewer: ok. done everywhere.

> If one has for instance a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, SOSS can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A<->B bridge is configured with a text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.

Author: ```suggestion
If one has, for instance, a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, SOSS can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A<->B bridge is configured with a text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.
```

Notice that more than two systems can be involved, as SOSS can be easily configured to bridge an arbitrary number of systems.

> The communication between these protocols is made possible by system-specific plugins, that we will refer to henceforth as System Handles (for detailed info, consul section ). These provide the necessary conversion between the target protocol and the specific language spoken by SOSS. Once a system is communicated with SOSS, it enters the SOSS world and can be straightforwardly communicated with any other system that already exists in this world.

Author: ```suggestion
The communication between these protocols is made possible by system-specific plugins, that we will refer to henceforth as System-Handle (for detailed info, consult section <insert_section_here>). These provide the necessary conversion between the target protocol and the specific language spoken by SOSS. Once a system is communicated with SOSS, it enters the SOSS world and can be straightforwardly communicated with any other system that already exists in this world.
```

In addition to the suggestions, this line is too long.

Reviewer: ok.

    :alt: eProsima
    :target: http://www.eprosima.com/

:code:`soss` is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that speak different languages.

> If one has, for instance, a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, :code:`soss` can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A⟷B bridge is configured with a YAML text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.

Author: Line extremely long.
Which other tools can be chosen instead of soss?
Is soss a message-passing protocol?
;P

    :alt: eProsima
    :target: http://www.eprosima.com/

:code:`soss` is a *System-Of-Systems Synthesizer* that allows communication among an arbitrary number of protocols that
speak different languages.

If one has, for instance, a pair of complex systems, A and B, and wills to combine them to create a larger, even more

> complex system, :code:`soss` can be chosen as an intermediate message-passing protocol that, by speaking a common

Author: Is soss a message-passing protocol?

Reviewer: changed protocol to tool.
